src/App.js

```diff
@@ -1,16 +1,32 @@
-import React from 'react';
+import React from "react";
+import ReactDom from "react-dom";
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      greeting: "Todo List: MVP",
+      Todo: ["ToDo"]
+    };
+  }
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
   render() {
     return (
       <div>
-        <h2>Welcome to your Todo App!</h2>
+        <h2>Todo List: MVP</h2>
+        <h1>Learn setstate()</h1>
+        <h1> Style my Todo list</h1>
       </div>
     );
   }
 }
 
+changeGreetingHandler = event => {
```

Comment:
@Fashoky you really don't need to be worried about the greeting handler for this project.  You should be focused on hard coding an array of state objects, and then passing state down as props to your TodoList.  Similar to how bands were passed down in the example code.